EventsExpress/Controllers/AuthenticationController.cs

```diff
@@ -152,19 +175,29 @@ public async Task<IActionResult> TwitterLogin([FromBody] UserView userView)
             {
                 UserDTO user = _mapper.Map<UserDTO>(userView);
                 user.EmailConfirmed = true;
```

Comment:
These lines of code are duplicated three times. Consider extracting them to a shared method.